ocaml/xcp-rrdd/bin/rrdp-dcmi/rrdp_dcmi.ml

```diff
@@ -64,7 +64,7 @@ let gen_dcmi_power_reading value =
   ( Rrd.Host
   , Ds.ds_make ~name:"DCMI-power-reading"
       ~description:"Host power usage reported by IPMI DCMI"
-      ~value:(Rrd.VT_Int64 value) ~ty:Rrd.Gauge ~default:true ~units:"W" ~min:0.
+      ~value:(Rrd.VT_Int64 value) ~ty:Rrd.Gauge ~default:true ~units:"W" ~min:1. ~max:65534.
```

Comment:
Here's something that confuses me. I'd like to know why did we set `~value:(Rrd.VT_Int64 value)` despite the default of min is a float type?

Reply:
Here's something that confuses me. I'd like to know why did we set ~value:(Rrd.VT_Int64 value) despite the default of min is a float type?

Reply:
The firmware can only report in multiples of 1 W, due to the integer type here and in the DCMI spec, but you are right that it may change in the future.

The 'min' can only have one type, and float is the most general one in this context, so that is what the API expects.

I've changed the type from VT_Int64 to VT_Float now to future proof it though (i.e. parse the watts as float). Although the DCMI spec site says there will be no further updates to the spec (and instead Redfish should be used, but that cannot be accessed from Dom0 as easily as IPMI), but you never know.